pkg/analytics/user.go

```diff
@@ -30,184 +23,49 @@ type Validated struct {
 // located in ~/.klotho/
 var analyticsFile = "analytics.json"
 
-func CreateUser(email string) error {
-
+func GetOrCreateAnalyticsFile() AnalyticsFile {
```

Comment:
This feels like it should return an `error`

Reply:
What would you suggest we do with that error?

The code as it stands:
1. Tries to get the current analytics.json file
2. If that errors (probably because it doesn't exist, but possibly for other reasons) then:
  1. create a new tracking id (in the form of the `AnalyticsFile` struct); this can never error
  2. try to write that id to analytics.json; this can error, in which case we log a debug and proceed
  3. send that newly created tracking id up to the caller

If we returned an error from this method, I could imagine doing one of two things with it:
1. logging it and proceeding, in which case that's the same as this currently logic
2. erroring out to the user

Am I missing an option? If not, do you think we should do option 1 (in which case imo it's just a stylistic question) or 2 (in which case I disagree, but we should have that discussion)?

Reply:
Regardless of (1) or (2) it doesn't feel like the analytics package should be the "decider" of that behaviour - so I guess it's a stylistic matter.